app/controllers/nonresident/CalculatorController.scala

```diff
@@ -122,6 +123,60 @@ trait CalculatorController extends BaseController {
 
     Future.successful(Ok(Json.toJson(result)))
   }
+
+  def useRebasedCalculation(rebasedValue: Option[Double], rebasedCosts: Option[Double], improvementsAft: Option[Double]): Boolean = {
+    Seq(rebasedValue, rebasedCosts, improvementsAft).forall(_.isDefined)
+  }
+
+  def useTimeApportionedCalculation(disposalDate: Option[DateTime], acquisitionDate: Option[DateTime]): Boolean = {
```

Comment:
I can see this code has been refactored out to it's own function because it **can** be reused, but at present it isn't so no need to pull it out.